msgraph-sdk-java.yml

```diff
@@ -0,0 +1,38 @@
+page_type: sample
+products:
+  - office-excel
+  - office-365
+  - office-outlook
+  - office-onedrive
+  - office-onenote
+  - office-planner
+  - office-sp
+  - ms-graph
+languages:
+  - java
+extensions:
+  contentType: sdks
+  technologies:
+    - Microsoft Graph
+    - Microsoft identity platform
+  services:
+    - Education
+    - Excel
+    - Groups
+    - Insights
+    - Intune
+    - Microsoft identity platform
+    - Office 365
+    - Outlook
+    - OneDrive
+    - OneNote
+    - Planner
+    - Reports
+    - Security
+    - SharePoint
+    - Users
+  platforms:
+    - Android
```

Comment:
Add Java as a platform as well.

Reply:
We combine languages and platforms in the gallery. Since it's included under languages, if we list it twice, it will be duplicated.